libs/platform/src/lib/components/menu-button/menu-button.component.html

```diff
@@ -6,10 +6,9 @@
     [label]="label"
     [compact]="contentDensity === 'compact'"
     [glyph]="icon"
-    [disabled]="disabled"
```

Comment:
Why did you remove `[disabled]` button state?
You are trying to move this `is-disabled` class to the host element itself, but I believe it's not safe.
`.fd-button.is-disabled` has some css overrides but in this case you miss it.
Let's avoid it and try to find another way.

Reply:
Also in this case you are introducing A11 issue as well.

Reply:
keeping [disabled] now.
needed to disabled cursor/pointer-event at menu-button level, to avoid click when disabled. corrected it now.